### ISSUE TITLE
replay

### ISSUE THREAD

**Issue by TheCherry** [openai/random-network-distillation#7]
The replayer.py use a exptag. I can't find this library.

**Comment by michael20at** [comment]
I have the same issue, exptag seems to be missing.

**Comment by michael20at** [comment]
You can kinda get it to work if you supply the temp directory with the pk file directly in the replayer.py file, commenting out the exptag line and import. Still, a proper solution would be nice!

**Comment by TheCherry** [comment]
Yes got it working, I will make a pull request soon. https://www.youtube.com/watch?v=OcNg6hvZB58
